app/_locales/zh_TW/messages.json

```diff
@@ -873,6 +1048,12 @@
   "showHexDataDescription": {
```

Comment:
Missing translation
```json
"show": {
  "message": "顯示"
}
```

Reply:
fixed in 58912402ec705258789f6da9e2fabfad980e929b